<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lblSortMethod">
              <property name="text">
               <string>Metoda de sortare:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbSortMethod">
              <item>
               <property name="text">
                <string>QuickSort</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Selection Sort</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bubble Sort</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Insertion Sort</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Merge Sort</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lblArraySize">
              <property name="text">
               <string>Dimensiunea sirului:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbSize">
              <property name="minimum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pbShuffle">
            <property name="text">
             <string>Amesteca</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbPause">
            <property name="text">
             <string>Intrerupe sortarea</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbResume">
            <property name="text">
             <string>Reia sortarea</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
